src/components/HomePage/SearchResultsArea/SearchResultsWidget/SearchURLWidget.js

```diff
@@ -40,6 +40,7 @@ const SearchURLWidget = () => {
     const classes = useSearchURLWidgetStyles();
     const dispatch = useDispatch();
 
+    // TODO: See if this should come from the window browser object or from the router
```

Comment:
not for this PR?

Reply:
This PR is just about the location service, but since this could be tackled in the future i just left the TODO there. IDEs like to highlight these so someone may be interested :)

Reply:
Sure, you might want to track with an issue too